src/ConnectedMode.UnitTests/Binding/CSharpVBBindingConfigProviderTests.cs

```diff
@@ -125,8 +125,8 @@ public async Task GetConfig_NoSupportedActiveRules_ReturnsNull()
             result.Should().BeNull();
 
             builder.Logger.AssertOutputStrings(1);
-            var expectedOutput = string.Format(Strings.SubTextPaddingFormat,
-                string.Format(Strings.NoSonarAnalyzerActiveRulesForQualityProfile, validQualityProfile.Name, Language.VBNET.Name));
+            var expectedOutput = string.Format(BindingStrings.SubTextPaddingFormat,
```

Comment:
I added a new resource file called `BindingStrings` to the `ConnectedMode` assembly and add the required strings to it (surprisingly only two).